cmds/modules/provisiond/main.go

```diff
@@ -365,6 +365,7 @@ func action(cli *cli.Context) error {
 	// call the runtime upgrade before running engine
 	if err := provisioners.Initialize(ctx); err != nil {
 		log.Error().Err(err).Msg("failed to run provisioners initializers")
+		return err
```

Comment:
i know it make sense, but it was like that for a long time. are we sure we need to return here

Reply:
yes I discussed it with azmy and ashraf and ashraf agreed we will enforce retry here for now